src/configuration/configuration.cpp

```diff
@@ -67,6 +67,16 @@ DEFINE_string(socket_family,
               "AF_INET",
               "Socket family (default: AF_INET)");
 
+DEFINE_string(private_key_file,
+              // TODO: change later
+             "/home/vagrant/temp/server.key",
+             "path to private key file");
+
+DEFINE_string(certificate_file,
+              // TODO: change later
+             "/home/vagrant/temp/server.crt",
```

Comment:
Indeed, this line should be changed. What if the user doesn't use vagrant? It's be better to find a way to access a directory inside the peloton project; I don't know what the correct approach is. I would try different paths in the code ("/peloton/your/path/to/file", "../your/path/to/file", "./your/path/to/file", etc.); if none of them works, I would give the user a warning.